### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by hansvancalster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,5 @@ Even geduld, mee bezig.
 Nu heb ik het pas door denk ik
 
 [fruits](./data/fruits.csv)
+
+In deze repository wil ik graag deze zin toevoegen
```
Comment (damianooldoni) [inline README.md]: Please, add a while line at the end of the text

File: README.md [hunk 2]
```diff
@@ -19,3 +19,5 @@ Even geduld, mee bezig.
 Nu heb ik het pas door denk ik
```
Comment (damianooldoni) [inline README.md]: Could you write it in English?

**Review by damianooldoni — CHANGES_REQUESTED** [review]
Good work. Just little changes to make this `readme.md` a great one!

File: README.md (hunk 2 above)
Comment (hansvancalster) [reply README.md]: will do :smile:

Comment (damianooldoni) [reply README.md]: :gb:  thanks

**Pull request merged**